index.html

```diff
@@ -14,11 +14,13 @@
     </style>
   </head>
 
-<script src="/socket.io/socket.io.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/socket.io/2.3.0/socket.io.js"></script>
+<script type="text/javascript" src="/bariobare.js"></script>
 <script src="https://code.jquery.com/jquery-3.4.1.min.js"></script>
   <script>
-  $(function () {
-    var socket = io();
+
+  function initialize(socketURL) {
+    var socket = socketURL ? io(socketURL) : io();
```

Comment:
Initializing the socket.io library by doing `io()` makes it try to connect to `window.location`, which is the URL in the address bar. The bariobare server might launch an instance of your game server at a different URL, so we need to pass it in as a parameter.